db/migrate/20151002192317_remove_books_from_books.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveBooksFromBooks < ActiveRecord::Migration
+  def change
+    remove_column :books, :books, :string
```

Comment:
I bet that got added because we did `rails g migration AddAttributesToBooks books ...`. The migration generator automatically figures out the table name from the migration name, so you don't have to add "books" as a parameter after it.